blogposts/2020/announcing-sourcegraph-3.16.md

```diff
@@ -0,0 +1,232 @@
+</p>
+-->
+
+Many large organizations have old versions of code running in production and need to search across all the code for that release. Sourcegraph 3.16 introduces the ability to create a context consisting of many repositories with specific revisions to be searched. As you explore search results, you might need to navigate your code. Version contexts are respected by code intelligence hovers (go to definition and find references), and allow you to quickly traverse the specific code revisions you care about.
+
+In this release, any non-master branches that are specified in the context will use our non-indexed search. This means that results may be slow for larger deployments. We are working on adding support for indexing additional branches that are specified within the context, and will include this in a future release.
+
+To take advantage of of this feature, add a `versionContexts` block to the experimental features in your site configuration:
+
+```json
+"experimentalFeatures": [
+  "versionContexts": [
+    {
+      "name": "srcgraph 3.16",
+      "revisions": [
+        {
+          "repo": "github.com/sourcegraph/sourcegraph",
+          "ref": "v3.16.0"
+        },
+        {
+          "repo": "github.com/sourcegraph/zoekt",
+          "ref": "b56036a3b745033badc48807bd67e91cd8d73bdf"
+        }
+      ]
+    }
+  ]
+]
+```
+
+## Campaigns
+
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://www.youtube.com/embed/aqcCrqRB17w" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://youtu.be/aqcCrqRB17w" target="_blank">View on YouTube</a></p>
+</p>
+
+Campaign changeset usage counts including changesets created, added, and merged will be sent back in pings. [#10591](https://github.com/sourcegraph/sourcegraph/pull/10591)
+
+## Best in class syntax highlighting
+
+TODO: Erik is creating screenshots
+
+We have made major improvements to syntax highlighting throughout Sourcegraph. In addition to general improvements, six new languages are supported (Smarty, Ethereum / Solidity / Vyper, Cuda, COBOL, vb.NET, and ASP.NET), and 30 new file extensions are now detected. 
+
+Additionally, this release adds best-in-class syntax highlighting to code diffs. This richer view of diffs can easily be accessed from your pull request (or merge request) using the Sourcegraph browser extension by clicking on the Sourcegraph button in a file header. Alternatively, you can compare revisions directly from Sourcegraph by going to the compare section from a repository page. You can also see this in action on your [campaigns](#campaigns) to make changes across all the repositories at your organization. 
+
+Take a look at the syntax highlighting on a diff that &lt;WHAT IS THIS DIFF ABOUT> in the Sourcegraph repository.
```

Comment:
@eseliger - can you find a link to nice diff that shows off syntax highlighting to include here?

Reply:
The link to the above screenshots is https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/commit/3fa28d025d615894867f550dfdbf5fb5b6de958f

Reply:
Added GIF that transitions between light and dark mode Sourcegraph diffs